CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.0.1.dev25 (2018-09-20)
+-----------------------
+
+Removed
+~~~~~~~
+* Removed superfluous jsonschema validation logging.
```

Comment:
Should that be 'JSON schema'?

Reply:
The library/package is called `jsonschema` https://pypi.org/project/jsonschema/

It's the external library that creates the error message, not the idea of having a schema for JSON.